python.html

```diff
@@ -129,14 +127,14 @@
 %% js
 // javascript
 var square = pyodide.pyimport("square")
-square([2.5], {integer: true})
+square(2.5)
```

Comment:
up above on [L:125](https://github.com/iodide-project/iodide-examples/pull/38/files#diff-c82d0dddcd762c99c5ab6c42dd31bd59R125), it states that js requires two args. Is this still the case? the python function works in the js side when you can the square function now